build-system/tasks/e2e/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
-  "name": "gulp-e2e",
+  "name": "amp-e2e",
   "version": "0.1.0",
-  "description": "Gulp e2e",
+  "description": "amp e2e",
```

Comment:
I'm seeing that the E2E tests are now complaining about not being able to find `babel-regenerator-runtime` during CI. (See [logs](https://app.circleci.com/pipelines/github/ampproject/amphtml/4806/workflows/aaa18870-cf39-41c9-b487-2d65d195bdb3/jobs/63417/parallel-runs/0/steps/0-109).) 

![image](https://user-images.githubusercontent.com/26553114/111525001-7ae76900-8733-11eb-995a-a967340539d8.png)

This PR doesn't actually change any dependencies apart from removing `gulp`, so I'm not sure what's going on here. I'm pretty sure renaming the package isn't the cause. (Maybe an undeclared transitive dependency that used to be a part of `gulp` but has now gone away?)

@estherkim Do you have any idea what's going on here?

Reply:
Can you try `npm install --loglevel verbose` in the e2e task to make sure it finishes before the runner moves on to the next step? Is that what you meant during standup?

Reply:
I was able to confirm that the install step was successful: https://app.circleci.com/pipelines/github/ampproject/amphtml/4792/workflows/ad5e2304-2efc-4d33-8f00-1863d54268df/jobs/63164/parallel-runs/0/steps/0-109

My current suspicion is that the cwd has somehow changed. I'll try to repro locally and fix. 

And yes, I plan on making it so that the install output is silenced when successful but prints errors during failures. NPM makes this surprisingly hard. https://github.com/npm/npm/issues/2040

Reply:
Great. That was my hunch.